### PR TITLE
Updating website for 0.5 and other changes

### PR DESCRIPTION
**Pull request by pksrc**
## Summary

Please describe your changes and what you plan to introduce as part of this pull request submission. If it fixes a bug or resolves a feature request, be sure to link to the issue numbers following the template below.

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [ ] Please ensure that you are making a pull request against the **Development** branch
- [x] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [ ] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [x] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commmits related to your change
- [x] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [x] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation changes
- [ ] Other (please describe):

## Testing Verification

* Tested on https://veba.pdotk.com/resources 

## Additional Information

CNAME will be changed during the final rebase after LGTM from reviewers

If you have any questions/comments, feel free to reach out to team on Slack [#vcenter-event-broker-appliance](https://vmwarecode.slack.com/archives/CQLT9B5AA)

Thank you from the VEBA Team! 🥳

### CODE REVIEW

**Comment by rguske** [comment]
Hi @pksrc, I'd assume that @embano1 will ask you to `squash` into just one final `commit`. If you don't mind, could you please do this?
Thanks buddy

**Comment by pksrc** [comment]
Updated with the latest Resources and the featured functions on the homepage has also been updated. 
Can be tested at https://veba.pdotk.com/resources & https://veba.pdotk.com

cc @rguske @vladi-velikov @kremerpatrick @embano1 @lamw - please verify content

All Commits will be squashed and the CNAME change reverted after a LGTM.

**Comment by kremerpatrick** [comment]
LGTM except the language for the Infrastructure Self Service + Automatic Remediations under featured functions. 'Additionally, enables you to handle certain crises are automatically to increase your operational efficiency' I'm thinking the stray 'are' needs to be removed?

**Comment by rguske** [comment]
@pksrc - you can delete the link to the `HA restart function` post under the `Other references` section, because it's already mentioned under `Featured References`. Thanks

**Comment by rguske** [comment]
Good catch @kremerpatrick - I fixed this in behalf of @pksrc while working on other updates and additions. Thanks

**Comment by embano1** [comment]
Note, in order to get these into the 0.5 release these changes (PR) must be made against the release branch.

File: docs/_functions/sample2.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Send VM Configuration changes to Slack 
+title: vSphere HA Event Notification Function
 icon: case-study-1.svg
 #subtitle: See Function
 links: 
- Deploy Function: https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/master/examples/powercli/hwchange-slack
+ Deploy Function: https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/development/examples/powercli/ha-restarted-vms
```
Comment (rguske) [inline docs/_functions/sample2.md:6]: The link has to reference to .../tree/`master`/examples...

**Review by rguske — CHANGES_REQUESTED** [review]
This little issue has to be fixed (commented): docs/_functions/sample2.md

**Pull request merged**